src/components/ClickablePicture.js

```diff
@@ -0,0 +1,32 @@
+import React from "react"
+import '../components/ClickablePicture.css'
+
+class ClickablePicture extends React.Component {
+
+    state={
+        clickedImg : false,

```

Comment:
This is looking good, I would just change this to true to make a picture without glasse by default :)